packages/adblocker-electron-preload/preload.ts

```diff
@@ -47,7 +50,22 @@ if (window === window.top && window.location.href.startsWith('devtools://') ===
       },
     );
 
-    getCosmeticsFilters({ lifecycle: 'start', ids: [], classes: [], hrefs: [] });
+    const scripts = getCosmeticsFiltersFirst();
+    if (scripts) {
+      const elems: HTMLScriptElement[] = [];
+      try {
+        scripts.forEach((script) => {
+          const e = document.createElement('script');
+          e.appendChild(document.createTextNode(script));
+          (document.head || document.documentElement || document).appendChild(e);
+          elems.push(e);
+        });
+      } catch (ex) {}
```

Comment:
```suggestion
      } catch {
        // continue regardless of error
      }
```

Seems CI is failing because of an empty catch without a comment
https://eslint.org/docs/latest/rules/no-empty